commands.go

```diff
@@ -66,6 +67,26 @@ func (w *commandWrapper) flush() {
 	w.out(true)
 }
 
+/*
+GetCommandFromString extracts command path from single string
+containing full path to command and arguments.
+*/
+func GetCommandFromString(command string) (string, error) {
```

Comment:
Unless go has helpers just get the first word from the string rather then trying to be clever. The support for commands with spaces in them is impossible to end well (unless you enforce escaping).